drydock/templates/drydock/k8s/jobs.yml

```
@@ -0,0 +1,113 @@
+{% for job in get_init_tasks() %}
+{{ job }}
+---
+{% endfor %}
+{% if MIGRATE_FROM in range(13,16) %}
```

This validation is kind of tricky in this template I think. Would it be OK if we set `MIGRATE_FROM` by default to `0` and change the validation to:
```suggestion
{% if MIGRATE_FROM %}
```
Anyways, inside every migration job you define which operations to run based on the `MIGRATE_FROM`. If a value that does not make sense is set, in the worst-case scenario, there will be a job that does not run anything